drgn/helpers/linux/net.py

```diff
@@ -260,3 +266,88 @@ def skb_shinfo(skb: Object) -> Object:
         return cast("struct skb_shared_info *", skb.head + skb.end)
     else:
         return cast("struct skb_shared_info *", skb.end)
+
+
+def skb_headlen(skb: Object) -> Object:
+    return skb.len - skb.data_len
+
+
+def skb_headroom(skb: Object) -> Object:
+    return skb.data - skb.head
+
+
+def skb_is_nonlinear(skb: Object) -> Object:
+    return skb.data_len
```

Comment:
I think it would make more sense for this one to return `bool`:

```suggestion
def skb_is_nonlinear(skb: Object) -> bool:
    return bool(skb.data_len)
```